eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -92,8 +93,10 @@ jobs:
       matrix:
         ${{ each matrixEntry in parameters.TestMatrix }}:
           ${{ if or(eq(matrixEntry.value.RunForPR, 'true'), ne(variables['Build.Reason'], 'PullRequest')) }}:
-            ${{ matrixEntry.key }}:
-              ${{ insert }}: ${{ matrixEntry.value }}
+            # Skip python version if any specific service has opted out
+            ${{ if not(contains(parameters.SkipPythonVersion, matrixEntry.value.PythonVersion)) }}:
```

Comment:
Why the contains check as opposed to exact string match?

Reply:
Reason for using contains is to support any future request of disabling more than one version.

Reply:
For now I suggest doing exact string match to ensure we don't hit any weird cases. Once we decide to support multiple versions I suspect we will need to do a little more logic like surrounding with some delimiter to ensure we don't match sub-strings.

Reply:
I have made this change to compare exact name. We can revisit this when we need to support multiple types. Thanks